polkadot-parachains/src/command.rs

```diff
@@ -41,8 +41,8 @@ use sp_core::hexdisplay::HexDisplay;
 use sp_runtime::traits::Block as BlockT;
 use std::{io::Write, net::SocketAddr};
 
-const LOCAL_PARA_ID: u32 = 2001;
-const ROCOCO_PARA_ID: u32 = 3002;
+const LOCAL_PARA_ID: u32 = 2015;
```

Comment:
Why do you change local as well? It's acceptable, but if you don' t use the launch script this means you have to register 15 empty paraids before getting the 2015 one